WcaOnRails/config/locales/en.yml

```diff
@@ -469,11 +469,6 @@ en:
           member: "Member"
           senior_member: "Senior Member"
           leader: "Leader"
-    person:
-      gender:
-        m: "Male"
-        f: "Female"
-        o: "Other"
```

Comment:
Why are you dropping these without replacement?

Reply:
This is not used anywhere now because we changed to the gender picker from wca-data.